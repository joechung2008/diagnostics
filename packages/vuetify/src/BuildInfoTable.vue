<script lang="ts" setup>
import type { BuildInfoProps } from './BuildInfoTable.types'

defineProps<BuildInfoProps>()
</script>

<template>
  <v-table>
    <thead>
      <tr>
        <th>Name</th>
        <th>Value</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>Build Version</td>
        <td>{{ buildVersion }}</td>
      </tr>
    </tbody>
  </v-table>
</template>
