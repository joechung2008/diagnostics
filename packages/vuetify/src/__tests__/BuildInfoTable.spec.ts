// BuildInfoTable.spec.ts
import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import BuildInfoTable from '../BuildInfoTable.vue'

describe('BuildInfoTable', () => {
  it('renders without crashing', () => {
    const wrapper = mount(BuildInfoTable, {
      props: {
        buildVersion: '1.0.0',
      },
    })
    expect(wrapper.exists()).toBe(true)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
