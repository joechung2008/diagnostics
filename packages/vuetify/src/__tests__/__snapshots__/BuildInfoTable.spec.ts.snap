// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BuildInfoTable > renders without crashing 1`] = `
"<div class="v-table v-theme--light v-table--density-default">
  <!---->
  <div class="v-table__wrapper">
    <table>
      <thead>
        <tr>
          <th>Name</th>
          <th>Value</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>Build Version</td>
          <td>1.0.0</td>
        </tr>
      </tbody>
    </table>
  </div>
  <!---->
</div>"
`;
